lib/miq_expression.rb

```diff
@@ -1212,8 +1213,10 @@ def self.reporting_available_fields(model, interval = nil)
       @reporting_available_fields[model.to_s] ||= {}
       @reporting_available_fields[model.to_s][interval.to_s] ||= MiqExpression.model_details(model, :include_model => false, :include_tags => true, :interval => interval)
     elsif Chargeback.db_is_chargeback?(model)
+      cb_model = Chargeback.report_cb_model(model)
       @reporting_available_fields[model.to_s] ||=
-        MiqExpression.model_details(model, :include_model => false, :include_tags => true).select { |c| c.last.ends_with?(*ReportController::Reports::Editor::CHARGEBACK_ALLOWED_FIELD_SUFFIXES) }
+        MiqExpression.model_details(model, :include_model => false, :include_tags => true).select { |c| c.last.ends_with?(*ReportController::Reports::Editor::CHARGEBACK_ALLOWED_FIELD_SUFFIXES) } +
+        MiqExpression.tag_details(cb_model, model, {})
```

Comment:
I think that `model_details` can return tags for you if to add `ChargebackVm` and `ChargebackContainerProject` tp `MiqExpression::TAG_CLASSES`